src/components/Elements/Form.js

```diff
@@ -4,6 +4,10 @@ import React, { Component } from 'react';
 import { reduxForm } from 'redux-form';
 import TextInput from '../../components/Form/TextInput';
 
+/**
+  * Renders a redux form that contains fields according to a `fields` config.
+  * @extends Component
+  */
 class Form extends Component {
```

Comment:
This may not apply to your branch, but is Form simple enough to be a function instead of extending Component?